Applications/Games/Steam/Online/script.js

```diff
@@ -10,6 +12,13 @@ var installerImplementation = {
             .url("https://steamcdn-a.akamaihd.net/client/installer/SteamSetup.exe")
             .checksum("4b1b85ec2499a4ce07c89609b256923a4fc479e5")
             .category("Games")
+            .wineVersion("4.8") // HOTFIX: https://github.com/PhoenicisOrg/scripts/issues/881
```

Comment:
Why not LATEST_STAGING VERSION ?

Reply:
Because currently `LATEST_STAGING_VERSION` is trying to install non-existant engine. (https://github.com/PhoenicisOrg/scripts/issues/881)

Once fixed i can use it.

Reply:
It will be fixed like today so do it already.

Reply:
doubt